build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```

Comment:
Changed ant script/installer scripts to just work, regardless of the username. Also, made it more jenkins friendly.